frontend/src/components/form/base/BaseComponent.vue

```diff
@@ -7,11 +7,12 @@
   >
     <component
       :is="inputComponent"
+      :name="name"
       v-bind="$attrs"
       :filled="filled"
       :hide-details="hideDetails"
       :error-messages="veeErrors.concat(errorMessages)"
-      :label.prop="label || name"
+      :label.prop="defaultLabel"
```

Comment:
This confused me at first, because I thought the label can't be overridden anymore. Could we call this labelOrDefault or similar instead?

Reply:
Good point. I now called it `labelOrEntityFieldLabel`